src/interfaces/IMorpho.sol

```diff
@@ -319,8 +326,14 @@ interface IMorphoStaticTyping is IMorphoBase {
 /// @dev Use this interface for Morpho to have access to all the functions with the appropriate function signatures.
 interface IMorpho is IMorphoBase {
     /// @notice The state of the position of `user` on the market corresponding to `id`.
+    /// @dev Warning: For `feeRecipient`, `supplyShares` does not contain the additional shares accrued since the last
+    /// interest accrual.
```

Comment:
```suggestion
    /// @dev Warning: For `feeRecipient`, `supplyShares` does not contain the accrued shares since the last interest
    /// accrual.
```
to have the same comment as on the struct (this is showing why it's not good practice to duplicate comments either)

Reply:
> to have the same comment as on the struct (this is showing why it's not good practice to duplicate comments either)

Even on an immutable contract?